### PR TITLE
Project : Positioning-and-floating-elements

### PR DESCRIPTION
**Pull request by sipe-daniel**
This project consists of making a clone of a New York Times article page.

### CODE REVIEW

**Comment by kobiyoyo** [comment]
## Pull Request under review, by Adama Chubiyojo Desmond (TSE)

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team   
#### Great Job so far, but there are few more things required. 

### Page structure
#### General
- [x]  Please lets fix this overlapping error(Editors’ Picks).

#### Main Content
- [x] The whole Content and **The Theory of Inflation** is not centered .
##### Original Version 



##### Your Version   


#### Footer
- [x]  Your page should also have this fixed footer that sticks at the bottom (check original page)



### Make changes and submit another code review request.
*Happy coding*

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -1,47 +1,770 @@
...
+    <link href="https://g1.nyt.com/fonts/css/web-fonts.5810def60210a2fa7d0848f37e3fa048bb6147b1.css" rel="stylesheet"
+        type="text/css">
+</head>
+
+<body>
+    <header>
+        <nav class="top">
+            <ul class="navigation">
+
+                <li class="menu"><i class="fa fa-bars"></i></li>
```
Comment (addod19) [inline index.html]: It has a hamburger button
hamb

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

Hi Team, you have done a great job. :100:  **Congratulations** but before we can approve this project, we need to fix the following: :slightly_smiling_face: 
- Kindly check the CSS validator errors [here](https://jigsaw.w3.org/css-validator)
ts

- Kindly fix the horizontal scroll
hrs

Kindly make these changes and submit code review :+1: 
Happy coding :computer:

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: great!
Just two things to fix before this project can be approved:
* [x]  Pull Request should have meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) Right now your title is not really meaningful, and description should have list of changes you implemented while working on this project.
* [x] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

File: css/style.css [hunk 2]
```diff
@@ -1,54 +1,963 @@
...
+    margin: 3px 0px 0px;
+    text-align: right;
+    color: rgb(153, 153, 153);
+    line-height: 1.23em;
+    font: inherit;
+    vertical-align: baseline;
     padding: 0;
+    border: 0;
+    float: right;
+} */
```
Comment (pelzolga123) [inline css/style.css]: This commented part should be removed.

File: index.html (hunk 1 above)
Comment (sipe-daniel) [reply index.html]: thanks for your observations

File: css/style.css (hunk 2 above)
Comment (sipe-daniel) [reply css/style.css]: Thanks for all.

**Review by addod19 — APPROVED** [review]
## Status: Approved :airplane: 

Hi **TEAM** you have really put in a great effort in this project. Nice work correcting the changes requested by reviewers.  I will have to approve this project.  :slightly_smiling_face: 
Before you merge to master kindly check and remove the **horizontal scroll**. It takes away the beauty and effort of the project :+1: 

Don't forget to submit a project completion form :100: 

Happy coding :computer:

**Pull request merged**